services-geojson/src/main/java/com/mapbox/geojson/utils/PolylineUtils.java

```diff
@@ -210,6 +214,10 @@ private static double getSqDist(Point p1, Point p2) {
 
   /**
    * Square distance from a point to a segment.
+   *
+   * @param point {@link Point} whose distance from segment needs to be determined
+   * @param p1,p2 points defining the segment
```

Comment:
Looking through this PR again, can we split up these javadoc params into two different tags? Each for p1 and p2. In fact, following our checkstyle rules, these params should be renamed to `pointOne` and `pointTwo` at the very least (or even a more descriptive name if possible).